jsonnetfile.lock.json

```diff
@@ -1,67 +1,71 @@
 {
```

Comment:
Do we want this in 4.4 version, if yes can you please update the deps in a separate commit so its easier to backport it to the release branch?

Reply:
The release-0.4 branch does not have this problem.